src/components/UserProfile/FoodTicket.tsx

```diff
@@ -0,0 +1,171 @@
+      queuePosition = "You are first in line!";
+    } else {
+      queuePosition = data?.queuePosition;
+    }
+    setFoodTicketData({
+      queuePosition: queuePosition,
+      eventName: data?.eventName,
+      eventDescription: data?.eventDescription,
+      eventTime: data?.eventTime,
+      timeslot: data?.timeslot,
+      userCode: shortenString(data?.verificationCode as string, 15),
+    });
+  }, [data]);
+
+  return (
+    <>
+      {isFetching ? (
+        <div className="flex h-screen w-full items-center justify-center bg-fuzzy-peach">
+          <h1 className="text-2xl">Loading...</h1>
+        </div>
+      ) : (
+        <div className="flex size-full flex-col bg-fuzzy-peach">
+          <div className="hidden md:block">
+            <Image
+              src="/images/userProfile/Star_Icons.svg"
+              alt="Right Squiggly SVG"
+              width={30}
+              height={30}
+              className="md:absolute md:left-10 md:top-[50rem]"
+            />{" "}
+            <Image
+              src="/images/userProfile/Star_Icons.svg"
+              alt="Right Squiggly SVG"
+              width={30}
+              height={30}
+              className="md:absolute md:right-8 md:top-[30rem]"
+            />{" "}
+            <Image
+              src="/images/userProfile/Star_Icons.svg"
+              alt="Right Squiggly SVG"
+              width={30}
+              height={30}
+              className="md:absolute md:right-10 md:top-[60rem]"
+            />{" "}
+          </div>
+          <div className="px-10 md:px-16 md:py-10">
+            <ProfileLinks />
+            <div className="mb-3 flex flex-col justify-between uppercase text-[#FF6B54] md:mx-10">
+              <div className="flex flex-row justify-between">
+                <h1 className="mt-3 text-lg font-bold md:text-2xl">
+                  My Food Ticket
+                </h1>
+              </div>
+              <div className="bg-pink mt-10 flex flex-col items-center gap-5 rounded-3xl border-4 border-white bg-white/30 px-8 py-20 sm:flex-row sm:justify-start lg:gap-12 lg:px-20 lg:py-16">
+                <QRCode
+                  value="https://www.google.ca" // placeholder
+                  size={300}
+                  logoImage="/images/userProfile/ctclogo.jpg"
+                  eyeColor={["#FF4D6F", "#FF4D6F", "#FF4D6F"]}
+                  eyeRadius={50}
+                  qrStyle="fluid"
+                  fgColor="#FF4D6F"
+                  style={{ borderRadius: "20px" }}
+                  logoPadding={3}
+                  logoPaddingStyle="square"
+                />
+                <div className="flex flex-col flex-wrap">
+                  {Object.entries(foodTicketData).map(([key, value]) => (
+                    <div className="flex w-full flex-wrap" key={key}>
+                      <h1 className="my-2 w-fit text-lg md:mt-2 md:text-xl">
+                        <strong>{key}:</strong>{" "}
```

Comment:
The keys have no spaces between them, showing results like this on the user profile page: 
QUEUEPOSITION
EVENTNAME,

would be nice if it was not a single word